nuspec/dnx-clr-win-x64.nuspec

```diff
@@ -5,7 +5,7 @@
         <version>0.0.1-pre</version>
         <authors>Microsoft</authors>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
-        <description>The .NET Runtime Environment for Desktop CLR on Windows x64</description>
+        <description>Microsoft .NET Execution environment for Desktop CLR on Windows x64</description>
```

Comment:
When last time we change .NET back to KRE, we decided to leave this part because it doesn't have any impact and we knew there would be another renaming.